app/policies/user_policy.rb

```diff
@@ -34,6 +34,14 @@ def permitted_attributes
     end
   end
 
+  def reports?
+    user.role.in? [User::COORDINATOR, User::ADMIN]
+  end
+
+  def download_report?
+    user.role.in? [User::COORDINATOR, User::ADMIN]
+  end
```

Comment:
Now that we have a reports controller, perhaps we can have a report policy?

It's permissions would be based solely on the user like you have here, there is no Report object to pass in.